pkg/backend/watch.go

```diff
@@ -50,12 +50,6 @@ func (b *backend) Watch(ctx context.Context, prefix string, revision uint64) (<-
 	result := make(chan []*proto.Event, resultChanLength)
 
 	// include the current revision in list
-	if revision < 0 {
```

Comment:
aim to ignore the range stream hack logic in etcd interface?

Reply:
`revision` is uint64， so the compare is always false and the branch is dead

> aim to ignore the range stream hack logic in etcd interface?

`revision` is uint64， so the compare is always false and the branch is dead